lib/src/conflicts.rs

```diff
@@ -1001,23 +1002,34 @@ pub async fn update_from_content(
     }
 
     let mut contents = simplified_file_ids.map(|_| vec![]);
+    let mut all_resolved = simplified_file_ids.map(|_| true);
```

Comment:
@scott2000 I am not sure if we prefer add another "array" here or do we prefer introduce another struct and put all the information(`FileId`, contents, and resolved) in the struct, so that we don't need `zip`?

Reply:
Done. Removed `all_resolved`.